Automattic/wp-calypso
# Enable vertical question and generated design picker steps on horizon environment

#### Changes proposed in this Pull Request

This PR enables the environment variables `signup/site-vertical-step` and `signup/design-picker-generated-designs` true by default in the horizon environment. The first flag enables the vertical question screen, and the second flag enables the generated design picker screen during customer onboarding.

Read here on how we use horizon: PCYsg-5bh-p2

#### Testing instructions



This is tricky to test as it only impacts a specific environment, but it would be helpful to verify that these two flags are only default to true in the following environments: `development` & `horizon`. For the rest of the environment, we will still need to explicitly use feature flags.

Post-merge testing will be needed to ensure the changes work as expected.



Related to #64226
